misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-sap/src/main/java/de/metas/camel/externalsystems/sap/api/model/product/ProductRow.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * #%L
+ * de-metas-camel-sap
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.camel.externalsystems.sap.api.model.product;
+
+import lombok.Getter;
+import org.apache.camel.dataformat.bindy.annotation.CsvRecord;
+import org.apache.camel.dataformat.bindy.annotation.DataField;
+
+@CsvRecord(separator = "	", skipField =true)
+@Getter
+public class ProductRow
+{
+	@DataField(pos = 1)
+	private String materialCode;
+
+	@DataField(pos = 3)
+	private String sectionCode;
+
+	@DataField(pos = 4)
+	private String name;
+
+	//FIXME: pending mappings
```

Comment:
is this still current? seems to me that you do use the value

Reply:
yep, this is still a thing as according to specs the UOM is in some custom format => so we might need custom mappings